torch/serialization.py

```diff
@@ -1172,7 +1188,14 @@ def __str__(self):
         return f'StorageType(dtype={self.dtype})'
 
 
-def _load(zip_file, map_location, pickle_module, pickle_file='data.pkl', **pickle_load_args):
+def _load(
+    zip_file,
+    map_location,
+    pickle_module,
+    pickle_file='data.pkl',
+    _default_load_endian=DefaultLoadEndian.LITTLE,
```

Comment:
Hey @AlekseiNikiforovIBM thanks for these additions. Could we 1) let this be a global variable instead  and 2) use the default `DefaultLoadEndian.Native` so this is BC-compatible, see [this comment on the initial issue](https://github.com/pytorch/pytorch/issues/101688#issuecomment-1583242312) for a longer discussion and let me know your thoughts on this :)